src/components/Calculator.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import CalculatorDisplay from './CalculatorDisplay';
+
+const Calculator = () => (
+  <>
+    <CalculatorDisplay />
+  </>
+);
```

Comment:
- [ ] Great job implementing this function using the ES6 syntax but the requirement states that you should use functional components. Don't worry, there will be time to refactor your code to make it up to date with the industry trend.
![image](https://github.com/chiedozie013/math-magicians/assets/99715640/c929a1ae-2855-41d7-9c59-a4ee75af90fb)